stdlib/Profile/src/Allocs.jl

```diff
@@ -59,18 +61,40 @@ function _prof_expr(expr, opts)
     end
 end
 
+"""
+    Profile.Allocs.start(sample_rate::Number)
+
+Begin recording allocations with the given sample rate
+A sample rate of 1 will record everything; 0 will record nothing.
+"""
 function start(; sample_rate::Number)
     ccall(:jl_start_alloc_profile, Cvoid, (Cdouble,), Float64(sample_rate))
 end
```

Comment:
```suggestion
"""
    Profile.Allocs.start(sample_rate::Real)

Begin recording allocations with the given sample rate
A sample rate of 1.0 will record everything; 0.0 will record nothing.
"""
function start(; sample_rate::Real)
    ccall(:jl_start_alloc_profile, Cvoid, (Cdouble,), Float64(sample_rate))
end
```

I like the way you used decimal numbers in the manual section; i think that helps to clarify the expected types of values to go here. 👍 Also, i realized that `Real` is probably the best type of number to expect here; it'd be weird nonsense to pass a Complex number here. :P And also `Real` suggests passing in decimal numbers more than `Number` does, so i think it is better documentation